dev/create-release/known_translations

```diff
@@ -416,3 +416,94 @@ zsxwing - Shixiong Zhu
 zuotingbing - Tingbing Zuo
 zuxqoj - Shekhar Bansal
 zzcclp - Zhichao Zhang
+Alfonso - Alfonso Buono
```

Comment:
Shall we sort them in alphabetical order?

Reply:
The file is an output of `translate-contributors.py` which just adds new items to the end of the file.